src/codegen/dotnet.ml

```diff
@@ -1204,14 +1203,28 @@ let add_net_lib com file std =
 		build path;
 		match !cp with
 			| [] -> None
-			| cp -> Some (!real_file, (pack,cp))
+			| cp -> Some (file_path, (pack,cp))
+
+	method get_data = ()
+
+	initializer
+		if std then self#add_flag FlagIsStd
+end
+
+let add_net_lib com file std =
+	let real_file = if Sys.file_exists file then
+		file
+	else try Common.find_file com file with
+		| Not_found -> try Common.find_file com (file ^ ".dll") with
+		| Not_found ->
```

Comment:
This is hard to read. At first sight I got "Why two `Not_found` cases"?

Reply:
I just copied that code. Feel free to clean up!